### DISCUSSION TITLE
Any clarification for listing iOS nearby WiFi SSIDs

### DISCUSSION THREAD

**Discussion by elmar001** [alternadom/WiFiFlutter#300] [Q&A]
I am just wondering if any explanation for this. How come native iOS app with location and access wifi permissions easily can list nearby WIFI SSIDs but it is not possible with Flutter. Pretty sure native iOS app which I tested myself does not have NEHotspotHelper API approval. Is this really limitation by iOS system or what is the case with Flutter?

**Comment by ghost** [comment]
Late response, but I'm pretty sure it's not a limitation by Flutter or iOS per se. `NEHotspotHelper` is ONLY for finding nearby hotspots (i.e. to get internet) and Apple requires you to have that use-case. You cannot use it for other purposes.
